include/binlog/adapt_stdtimepoint.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef BINLOG_ADAPT_STDTIMEPOINT_HPP
+#define BINLOG_ADAPT_STDTIMEPOINT_HPP
+
+#include <mserialize/cx_string.hpp>
+#include <mserialize/serialize.hpp>
+#include <mserialize/tag.hpp>
+
+#include <chrono>
+#include <cstdint>
+
+namespace mserialize {
+
+template <>
```

Comment:
If the user was to use `std::chrono::time_point_cast` on the `time_point` before passing it to binlog then it would not be serialized. Can we support a 
```
template<typename TDuration>
std::chrono::time_point<std::chrono::system_clock, TDuration>
```
type?

Reply:
Good idea, fixed.